src/briefcase/platforms/iOS/xcode.py

```diff
@@ -370,10 +370,12 @@ def _install_app_requirements(
             pip_args=[
                 f"--platform=ios_{ios_min_tag}_arm64_iphoneos",
             ],
+            install_hint="It may also indicate that an `iphonesimulator` wheel could be found, "
+            "but an `iphoneos` wheel could not be found.",
```

Comment:
As above, ordering of arguments needs tweaking. There's also a need to add extra whitespace at the start:
```suggestion
            install_hint="""

It may also indicate that an `iphonesimulator` wheel could be found, but an
`iphoneos` wheel could not be found.
"""
```